glib/src/utils.rs

```diff
@@ -200,8 +178,6 @@ pub fn tmp_dir() -> std::path::PathBuf {
 pub fn mkstemp<P: AsRef<std::path::Path>>(tmpl: P) -> i32 {
```

Comment:
This one has to stay manual, I think all others can be autogenerated now

Reply:
I don't understand

Reply:
In `Gir.toml` most of the functions affected here are marked as `manual = true` or `ignore = true`. That causes them to not be autogenerated in `glib/src/auto/functions.rs`.

All functions except for `g_mkstemp` can be autogenerated now. You'd remove them from `Gir.toml` so that the `manual = true` is gone, then regenerate, then they end up in `functions.rs` and you can remove the manual implementation from `glib/src/utils.rs`.

Reply:
okay. let me check. thanks

Reply:
But also all functions except for `mkstemp()` can be autogenerated now, as said before.